CMakeLists.txt

```diff
@@ -276,15 +277,15 @@ if(CMAKE_SYSTEM_NAME STREQUAL "OS400")
 endif()
 
 if(SENTRY_TRANSPORT_CURL)
-	if(NOT CURL_FOUND) # Some other lib might bring libcurl already
+	if(NOT TARGET CURL::libcurl) # Some other lib might bring libcurl already
```

Comment:
There is no problem with calling `find_package` repeatedly. However, you are really interested in `CURL::libcurl` already being created, e.g. by a subproject.